backend/corpora/common/corpora_orm.py

```diff
@@ -30,7 +31,7 @@ class TransformingBase(object):
     """
 
     def __iter__(self):
-        return self.to_dict().iteritems()
+        return iter(self.to_dict().items())
```

Comment:
what is the difference?

Reply:
I believe `iteritems` isn't a thing python3

Reply:
ahhh